ui/src/common/playingInAlbumOrPlaylist.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types'
+
+//songId is the ID of the playlist or album corresponding to the song
+//activeId is the ID of the current album or playlist
```

Comment:
You can use JSDoc style of commenting to give description about the variable because then VSCODE Instellesence would pick it up too. 

```
/**
 *  // Description of the function
 * @param {string} songID - the ID of the playlist or album corresponding to the song 
 * @param {string} activeID - the ID of the current album or playlist - the ID of the playlist or album corresponding to the song 
 * @returns {boolean} 
 */

```

Reply:
Thanks, I'll implement this :)